scripts/styleguide.config.js

```diff
@@ -97,6 +100,9 @@ const allSections = [
             '../src/components/toggle/Toggle.js',
             '../src/components/tooltip/Tooltip.js',
         ],
+        description: 'Box UI Elements components implement the reusable building blocks of the Box Design Language',
+        sectionDepth: 2,
```

Comment:
This makes each component have its own route instead of one huge page with every component